packages/expo-updates-interface/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Removed deprecated Android `kotlin-android-extensions` plugin ([#19732](https://github.com/expo/expo/pull/19732) by [@josephyanks](https://github.com/josephyanks))
```

Comment:
Same here

Reply:
Just calling out that I think this is in the right spot (others). Going to resolve but feel free to unresolve.